### PR TITLE
Add reportwidget Statistics

### PR DESCRIPTION
**Pull request by WebVPF**
Add reportwidget Statistics.

The widget displays information:
- number of registered users
- number of activated users
- number of banned users

The number of active and the number of registered users for today, yesterday, month, year is also displayed.

Screenshot:

DarkTheme screenshot:

### CODE REVIEW

**Comment by bennothommo** [comment]
@WebVPF I really like this idea, but I think it needs to look more like our other widgets and the overall Winter UI - minimalist and clean. I'd use less colour, and instead of the 3 abstract <div>'s with the main stats (registered users, activated users, banned users), you could perhaps use the [Scoreboard](https://wintercms.com/docs/ui/scoreboard) UI component to display these values.

Could you make these changes and do some screenshots of the English lang look as well?

File: lang/en/lang.php [hunk 1]
```diff
@@ -219,5 +219,27 @@
         'redirect_desc' => 'Page name to redirect if access is denied.',
         'logout' => 'You have been successfully logged out!',
         'stop_impersonate_success' => 'You are no longer impersonating a user.',
-    ]
+    ],
+    'reportwidgets' => [
+        'statistics' => [
+            'default_title' => 'Plugin User - Statistics',
```
Comment (bennothommo) [inline lang/en/lang.php:225]:
```suggestion
            'default_title' => 'User Statistics',
```

File: lang/en/lang.php [hunk 2]
```diff
@@ -219,5 +219,27 @@
...
+            'registration' => 'Registration',
+            'activated' => 'Activated',
+            'ban' => 'Ban',
+            'today' => 'today',
+            'yesterday' => 'yesterday',
+            'month' => 'month',
+            'year' => 'year',
```
Comment (bennothommo) [inline lang/en/lang.php:232]:
```suggestion
            'today' => 'Today',
            'yesterday' => 'Yesterday',
            'month' => 'This month',
            'year' => 'This year',
```

File: lang/en/lang.php [hunk 3]
```diff
@@ -219,5 +219,27 @@
...
+            'registrations' => 'Registrations',
+            'tooltip_registr_today' => 'Users registered today',
+            'tooltip_registr_yesterday' => 'Users registered for yesterday',
+            'tooltip_registr_month' => 'Users registered in last month',
+            'tooltip_registr_year' => 'Users registered in last year',
```
Comment (bennothommo) [inline lang/en/lang.php:237]:
```suggestion
            'tooltip_registr_today' => 'Users registered today',
            'tooltip_registr_yesterday' => 'Users registered yesterday',
            'tooltip_registr_month' => 'Users registered in the last month',
            'tooltip_registr_year' => 'Users registered in the last year',
```

File: lang/en/lang.php [hunk 4]
```diff
@@ -219,5 +219,27 @@
...
+            'activity' => 'Activity',
+            'tooltip_activity_today' => 'Active users today',
+            'tooltip_activity_yesterday' => 'Active users for yesterday',
+            'tooltip_activity_month' => 'Active users in last month',
+            'tooltip_activity_year' => 'Active users per year',
```
Comment (bennothommo) [inline lang/en/lang.php:242]:
```suggestion
            'tooltip_activity_today' => 'Active users today',
            'tooltip_activity_yesterday' => 'Active users yesterday',
            'tooltip_activity_month' => 'Active users in the last month',
            'tooltip_activity_year' => 'Active users in the last year',
```

**Review by bennothommo — CHANGES_REQUESTED** [review]

File: lang/en/lang.php (hunk 2 above)
Comment (WebVPF) [reply lang/en/lang.php:232]: @bennothommo you indicated how: this month, this year.
But that's not what I meant. Not this month / year, but the last 30/356 days.
That is, today is November 19, 2021.
`Month` - we take statistics from October 19 to November 19.
`Year` - statistics from November 19, 2020 to November 19, 2021

Comment (WebVPF) [reply lang/en/lang.php:232]: There is no error handling in my code. I'll fix it.

**Pull request closed without merging**